charsplit/__init__.py

```diff
@@ -0,0 +1,4 @@
+from charsplit.splitter import Splitter
+
+__author__ = 'don.tuggener@gmail.com'
```

Comment:
```suggestion
__author_email__ = 'don.tuggener@gmail.com'
```
you may add also `__author__` and `__version__` here :]